pkg/scheduling/volumeusage.go

```diff
@@ -97,37 +139,6 @@ func (v *VolumeUsage) Add(ctx context.Context, kubeClient client.Client, pod *v1
 	v.volumes = v.volumes.union(podVolumes)
 }
 
-type VolumeCount map[string]int
```

Comment:
Can you undo the file re-organization? It makes the diff larger and more difficult to review.

Reply:
Updated